### ISSUE TITLE
[Feature Request] improve docs

### ISSUE THREAD

**Issue by sansyrox** [sansyrox/robyn#202]
**Current Behavior**

Try using this https://squidfunk.github.io/mkdocs-material/
**Desired Behavior**

---
**Screenshots / Mockups**

**Alternatives**

---

**Comment by aryasoni98** [comment]
I would like to work on it .

**Comment by sansyrox** [comment]
Hi @aryasoni98 .

Thank you. Do let me know if you require any help from my end.

**Comment by aryasoni98** [comment]
@sansyrox I just need to improve the design part only or content part to ?

**Comment by aryasoni98** [comment]
@sansyrox can i use the hugo server theme ?

**Comment by sansyrox** [comment]
@aryasoni98 , I went through the docs of the link you shared with me. The search bar is only able to search through the headings but not the content. 

Do you know if it is possible to do that?

**Comment by sansyrox** [comment]
Also, is there a way to change the color scheme of the dark and white themes?

If both of these are possible then we can proceed with your suggestion . :D

**Comment by aryasoni98** [comment]
@sansyrox Yes, I can add the search bar in content header and also change the color scheme.

**Comment by sansyrox** [comment]
@aryasoni98 , perfect then. You may start working on the issue.

**Comment by aryasoni98** [comment]
> @aryasoni98 , perfect then. You may start working on the issue.

Okay Sure, Can you assign me.

**Comment by sansyrox** [comment]
Use docus: https://docus.dev/
